lib/davinci_dtr_test_kit/profiles/task/task_validation.rb

```diff
@@ -4,10 +4,10 @@ module DaVinciDTRTestKit
   class TaskValidationTest < Inferno::Test
     include DaVinciDTRTestKit::ValidationTest
 
-    title 'Task resources returned during previous tests conform to the CDex Task'
+    title 'Task resources returned during previous tests conform to the PAS Task'
     description %(
 This test verifies resources returned from the first read conform to
```

Comment:
first read -> read step

Reply:
Addressed in [first read/search->read/search step, input titles, wording](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/35/commits/898f1471140e63899323346b0d3296d48f7af929)